packages/turf-isobands/lib/marchingsquares-isobands.js

```diff
@@ -3,8 +3,38 @@
  * Copyright (c) 2015, 2015 Ronny Lorenz <ronny@tbi.univie.ac.at>
  * v. 1.2.0
  * https://github.com/RaumZeit/MarchingSquares.js
+ *
+ * MarchingSquaresJS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MarchingSquaresJS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * As additional permission under GNU Affero General Public License version 3
+ * section 7, third-party projects (personal or commercial) may distribute,
+ * include, or link against UNMODIFIED VERSIONS of MarchingSquaresJS without the
+ * requirement that said third-party project for that reason alone becomes
+ * subject to any requirement of the GNU Affero General Public License version 3.
+ * Any modifications to MarchingSquaresJS, however, must be shared with the public
+ * and made available.
+ *
+ * In summary this:
+ * - allows you to use MarchingSquaresJS at no cost
+ * - allows you to use MarchingSquaresJS for both personal and commercial purposes
+ * - allows you to distribute UNMODIFIED VERSIONS of MarchingSquaresJS under any
+ *   license as long as this license notice is included
+ * - enables you to keep the source code of your program that uses MarchingSquaresJS
+ *   disclosed
```

Comment:
should be 'undisclosed' (see https://github.com/RaumZeit/MarchingSquares.js/pull/25)